README.md

```diff
@@ -367,6 +368,16 @@ Vagrant.configure("2") do |config|
   end
 end
 ```
+#### Hyper-V
+```ruby
+Vagrant.configure("2") do |config|
+  # ...
+
+  c.vm.provider :virtualbox do |p|
```

Comment:
Looks like a typo s/:virtualbox/:hyperv/

Reply:
I've fixed it.